src/windows/mod.rs

```diff
@@ -529,6 +529,8 @@ extern "C" {
         category: ::c_int,
         locale: *const wchar_t,
     ) -> *mut wchar_t;
+    #[link_name = "_aligned_malloc"]
+    pub fn aligned_malloc(size_t size, size_t alignment) -> *mut c_void;
```

Comment:
The syntax is wrong.

Reply:
Oops I copied this straight from the docs... I fixed it now